recipes/prismatic/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+
+
+mkdir build && cd build 
+
+cmake -D PRISMATIC_ENABLE_GUI=1 \
+	-D CMAKE_INSTALL_PREFIX=$PREFIX \
+	../ 
+
+make
```

Comment:
```suggestion
make -j${CPU_COUNT}
```

Reply:
Yep - done (as part of a general tidy up commit)